### PR TITLE
Feature 69 Create pypi package

### PR DESCRIPTION
**Pull request by ami-day**
Created a setup.py file
Creaated an SCEA_curatoin_guidelines file: moved some of the guidelines from the README.md file to this new md file.
Created a new README.md file which is more suitable for a pypi package.
Added __init__.py files
Added requirements.txt file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from setuptools import setup, find_packages
+
+HERE = pathlib.Path(__file__).parent
+
+README = (HERE / "README.md").read_text()
+install_requires = [line.rstrip() for line in (HERE / "requirements.txt").read_text().splitlines()]
+
+setup(
+    name="hca-to-scea",
+    version="1.0.0",
```
Comment (jacobwindsor) [inline setup.py]: I think this should be v0.1.0. See [https://semver.org/](Semver versioning). New projects start at v0

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    description="A tool to assist in the automatic conversion of hca metadata to scea metadata MAGE-TAB files.",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/ebi-ait/hca-to-scea-tools",
+    author="Ami Day, Javier Ferrer, Yusra Haider, Alegria Aclan",
+    author_email="ami@ebi.ac.uk, javier.f.g@um.es, yhaider@ebi.ac.uk, aaclan@ebi.ac.uk",
```
Comment (jacobwindsor) [inline setup.py]: Typically this would be in order of who to contact with issues

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1 @@
+include requirements.txt
```
Comment (jacobwindsor) [inline MANIFEST.in]: You dont need to specify requirements.txt in the MANIFEST. So just remove it

File: README.md [hunk 4]
```diff
@@ -132,307 +94,95 @@ If accessions E-HCAD1 to E-HCAD32 have already been assigned to datasets, the ne
...
-We do not need to send SCEA the raw data files. The file paths should be in the sdrf file. They will use the filepaths to obtain the raw data.
+### Developing Code in Editable Mode
 
-**HCA curators**
+Using `pip`'s editable mode, projects using hca-to-scea as a dependency can refer to the latest code in this repository 
+directly without installing it through PyPI. This can be done either by manually cloning the code
+base:
 
-We now have a within-team secondary review process. Please move the dataset ticket to the new 'secondary review' column in the SCEA Zenhub Board and add a link to your idf and sdrf files. Please assign Ami as the secondary reviewer for now. This helps to highlight further areas for improvement of the hca2scea tool and the SOP.
+    pip install -e path/to/hca-to-scea
```
Comment (jacobwindsor) [inline README.md:155]: `pip install -e . ` is better

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: osx-64
+brotlipy=0.7.0=py39hcbf5805_1001
```
Comment (jacobwindsor) [inline requirements.txt]: How was this file created? I've not seen versioning with the py39... 

Also, please use less strict versioning. Typically we just version on major releases unless we know that we need a minor version for some reason. See https://www.python.org/dev/peps/pep-0440/#version-specifiers

**Review by jacobwindsor — CHANGES_REQUESTED** [review]
A few things but great job! Oh and please fix the merge conflict

**Review by ami-day — COMMENTED** [review]
Thanks Jacob, I have made the suggested changes except I need to ask you about the requirements versioning. I have a couple of other questions too, I'll message you on slack.

**Comment by ESapenaVentura** [comment]
Still working on this

**Review by jacobwindsor — CHANGES_REQUESTED** [review]
Have you tried installing this? `pip install .`

I get the error "ERROR: Could not find a version that satisfies the requirement sqlite (from hca-to-scea)
"

**Review by amnonkhen — CHANGES_REQUESTED** [review]
There seems to be a great mess here.
Many of these files should not be part of the PR, as they are files I have changed in a previous PR.
I need you to walk me through the steps you have taken to get here, as it appears that something went wrong on the way.

**Comment by amnonkhen** [comment]
> Created a setup.py file Creaated an SCEA_curatoin_guidelines file: moved some of the guidelines from the README.md file to this new md file. Created a new README.md file which is more suitable for a pypi package. Added **init**.py files Added requirements.txt file

No need for such comments as it is clear in the PR which file is added, modified or deleted.
It is more helpful to use informative sentences like "added package build scripts" or "update documentation with XYZ"

**Pull request merged**